Remotion/Data/DomainObjects/Queries/QueryManager.cs

```diff
@@ -97,7 +98,7 @@ public IClientTransactionEventSink TransactionEventSink
     {
       ArgumentUtility.CheckNotNull("query", query);
 
-      if (query.QueryType != QueryType.Scalar)
+      if (query.QueryType != QueryType.ScalarReadOnly && query.QueryType != QueryType.ScalarReadWrite)
```

Comment:
It's valid, but the meaning is wrong:

![image](https://github.com/re-motion/Framework/assets/58393218/6566b0e6-0a06-4af7-ba0e-2be053a139d4)
This prints "its ok.c"

since b is not a, it already sets it as true.

Reply:
thanks. means there's no shortcut and we always need to be careful with the pattern syntax if there's no parens used. unit tests for the win!